orion/sites/admin/src/achieve/achieveEdit.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+require('./achieveEdit.styl');
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Link } from 'react-router-dom';
+import API from '../api.js';
+import { Modal } from '../modals/modal.js';
+import { OkayModal } from '../modals/okayModal.js';
+import { YesNoModal } from '../modals/yesnoModal.js';
+
+export class AchieveEditPage extends React.Component {
+
+  render() {
+
+    return (
+      <div id="view-achieve-edit">
+        {modalDiv}
+        <h2>{title}</h2>
```

Comment:
For now, you can give all these variables `{...}` fake values to see which values relate to which fields.